Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -1,4 +1,4 @@
-#import "SentryLaunchProfiling.h"
+#import "SentryLaunchProfiling+Tests.h"
```

Comment:
`m`: I'm a bit confused. Why does production code include a test header?

Reply:
Mm, I thought I'd have to do this because I needed to expose some C declarations to tests, but I can just redeclare them in SentryLaunchProfiling.m as well and there doesn't seem to be an issue 👍🏻